### PR TITLE
Add background to thumbnails

### PR DESCRIPTION
**Pull request by korabh**

### CODE REVIEW

**Comment by korabh** [comment]
@Skud Please review this pull request.

File: app/assets/stylesheets/leaflet_overrides.css.less [hunk 1]
```diff
@@ -1,3 +1,10 @@
 .leaflet-popup-content-wrapper, .leaflet-popup-tip {
   border: none;
 }
+.thumbnail{
+	background: #ffff!important;
```
Comment (cesy) [inline app/assets/stylesheets/leaflet_overrides.css.less]: Why not either 3 or 6 "f"s? Can we also get a space before !important, or look to sort the CSS structure so it's not needed?

**Comment by cesy** [comment]
Can you add a description or screenshot of what this is intended to do, please, to help our testers? Please also have a look at the build failure.

**Comment by korabh** [comment]
Sure,

**Comment by cesy** [comment]
I like the background on the thumbnails, which emphasises the card-based design. I'm not so keen on the circles for user pics - I want mine square, the way I uploaded it, and get annoyed when websites override that.

**Comment by maco** [comment]
I'm with @cesy

**Comment by korabh** [comment]
I have change it to "img-rounded" :hammer:

**Comment by cesy** [comment]
Sorry to be a nuisance, but can we get a fresh screenshot, please, to save reloading the branch locally? Thanks.

**Comment by korabh** [comment]
rounded

**Comment by cesy** [comment]
That looks great, thanks. So now we just need to sort the build/test failure - it looks like it's broken something in the member bio pages.

**Comment by Skud** [comment]
Please don't use img-rounded.  We recently (a few months back?) 
standardised on using all square images, and removed any img-rounded 
instances that were in the code.  If there are any remaining, this is an 
error.

On 26/03/2015 2:06 am, Korab Hoxha wrote:
>
> I have change it to "img-rounded" :hammer:
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Growstuff/growstuff/pull/730#issuecomment-86072624>.
>

-- 
Alex "Skud" Bayley
skud@growstuff.org
http://growstuff.org/

**Comment by cesy** [comment]
Is the background still worth adding, to move us closer to the card-based view? @korabh, would you be up for removing the img-rounded part so we can do the rest? Sorry for the long delay on this.

**Comment by korabh** [comment]
@cesy Done.

**Comment by cesy** [comment]
Removing the space in the avatar file appears to be causing test failures where the page won't render. Can you check, please?

**Comment by korabh** [comment]
@cesy It's okay now

**Comment by cesy** [comment]
Excellent, thank you!

**Comment by korabh** [comment]
@cesy you're welcome

**Pull request merged**